pyvista/core/pointset.py

```diff
@@ -665,19 +685,19 @@ def __init__(
             raise TypeError(dedent(msg.strip('\n')))
 
         # At this point, points have been setup, add faces and/or lines
-        if faces is None and lines is None and strips is None:
+        if faces is None and lines is None and strips is None and verts is None:
```

Comment:
nits
```suggestion
        if faces and lines and strips and verts is None:
```

Reply:
I've never seen this type of excursion before.  I tested it and it works but it is unexpected to me.  When all are `None`, I would expect `verts is None` to evaluate to `True` and then `strips is verts is None` to evaluate to `False` because it compares `None` to `True`.  What am I missing?

Reply:
I knew this for knowledge but had never checked the Python documentation.

In the [Comparisons section of Python document](https://docs.python.org/3/reference/expressions.html#comparisons), we will find the following sentence. We can recognize `aces is None and lines is None and strips is None and verts is None` is equal `faces is lines is strips is verts is None` by this.

> Comparisons can be chained arbitrarily, e.g., x < y <= z is equivalent to x < y and y <= z, except that y is evaluated only once (but in both cases z is not evaluated at all when x < y is found to be false).

Reply:
This makes a lot more sense to me thanks!  I think the last example is very clear.  I think this is a bit more obscure to think about, whereas `x < y < z` is more natural.  I'm fine with the suggestion though, it is certainly shorter and compact.